cmd/mistryd/main.go

```diff
@@ -106,24 +104,23 @@ func main() {
 					Name:  "project, p",
 					Usage: "the project to build. Multiple projects can be specified. If not passed, all projects are built",
 				},
+				cli.BoolFlag{
+					Name: "verbose, v",
+				},
 			},
 			Action: func(c *cli.Context) error {
 				cfg, err := parseConfigFromCli(c.Parent())
 				if err != nil {
 					return err
 				}
 
-				logf, err := os.OpenFile(filepath.Join(cfg.BuildPath, "rebuild.log"), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+				logger := log.New(os.Stdout, "", 0)
+				r, err := RebuildImages(cfg, logger, c.StringSlice("project"), c.Bool("fail-fast"), c.Bool("verbose"))
 				if err != nil {
-					return err
+					return fmt.Errorf("Image error, existing early, partial result: %s", r)
```

Comment:
Perhaps a simpler message:

> Error building image: %s

Also, how come we don't print `err` here?

Reply:
The `Action` function returns an error or nil, and the caller takes care of printing the error if one is returned

Reply:
I mean, we don't use the `err` variable at all.